bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/UL_Radion_WW_inclu/UL_Radion_WW_inclu_narrow_M/UL_Radion_WW_inclu_narrow_M_proc_card.dat

```diff
@@ -0,0 +1,11 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
+import model heft_radion
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define ferm = u c d s b u~ c~ d~ s~ b~ e- mu- ta- e+ mu+ ta+ vl vl~
+generate p p > h2, ( h2 > w+ w-, w+ > ferm ferm , w- > ferm ferm)
```

Comment:
Do you may be need another parenthesis after the w + w- for the separate w decays?

Reply:
Hi, this card is mostly a copy & paste from an older [reference](https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/exo_diboson/Spin-0/Radion_WW_inclu_narrow/Radion_WW_inclu_narrow_M/Radion_WW_inclu_narrow_M_proc_card.dat) which does not use another parenthesis for separate W decays.